spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestExpireSnapshotsAction.java

```diff
@@ -1027,34 +1029,59 @@ public void testExpireAction() {
   @Test
   public void testUseLocalIterator() {
     table.newFastAppend()
-        .appendFile(FILE_A)
-        .commit();
+            .appendFile(FILE_A)
+            .commit();
 
     table.newOverwrite()
-        .deleteFile(FILE_A)
-        .addFile(FILE_B)
-        .commit();
+            .deleteFile(FILE_A)
+            .addFile(FILE_B)
+            .commit();
 
     table.newFastAppend()
-        .appendFile(FILE_C)
-        .commit();
+            .appendFile(FILE_C)
+            .commit();
 
     long end = rightAfterSnapshot();
 
     int jobsBefore = spark.sparkContext().dagScheduler().nextJobId().get();
 
-    ExpireSnapshots.Result results =
-        SparkActions.get().expireSnapshots(table).expireOlderThan(end).execute();
+    AtomicReference<Integer> totalJobsRun = new AtomicReference<>();
 
-    Assert.assertEquals("Table does not have 1 snapshot after expiration", 1, Iterables.size(table.snapshots()));
+    withSQLConf(ImmutableMap.of("spark.sql.adaptive.enabled", "false"), () -> {
+      ExpireSnapshots.Result results =
```

Comment:
The formatting is off in this entire file. Iceberg uses 2 spaces for identification and 4 spaces for continued identification (instead of 4 and 8 used in this file).

Reply:
Thanks for the explanation. Just updated the indention. :)